sw/device/lib/dif/dif_aes_unittest.cc

```diff
@@ -97,14 +129,17 @@ class EcbTest : public AesTestInitialized {
 
 TEST_F(EcbTest, start) {
   EXPECT_READ32(AES_STATUS_REG_OFFSET, 1);
-  SetExpectedConfig(AES_CTRL_SHADOWED_KEY_LEN_VALUE_AES_128,
-                    AES_CTRL_SHADOWED_MODE_VALUE_AES_ECB,
-                    AES_CTRL_SHADOWED_OPERATION_VALUE_AES_ENC,
-                    /*manual_op=*/false,
-                    /*force_zero_masks=*/false);
+  SetExpectedConfig({.key_len = AES_CTRL_SHADOWED_KEY_LEN_VALUE_AES_128,
+                     .mode = AES_CTRL_SHADOWED_MODE_VALUE_AES_ECB,
+                     .operation = AES_CTRL_SHADOWED_OPERATION_VALUE_AES_ENC,
+                     AES_CTRL_SHADOWED_PRNG_RESEED_RATE_VALUE_PER_1,
+                     .manual_op = false,
+                     .force_zero_masks = false,
+                     .key_sideloaded = false});
```

Comment:
Given that `false` is probably the default value, you can leave these off. I'd say you should only include "interesting" fields.

(Applies elsewhere.)

Reply:
Isn't it worth being explicit here to improve readability?

Reply:
I lean towards leaving them off. I don't think being explicit is that valuable when a feature is disabled.